wincred/wincred_windows.go

```diff
@@ -0,0 +1,40 @@
+package wincred
+
+import (
+	winc "github.com/danieljoos/wincred"
+	"github.com/docker/docker-credential-helpers/credentials"
+)
+
+type wincred struct{}
+
+// New creates a new wincred.
+func New() credentials.Helper {
+	return wincred{}
+}
+
+// Add adds new credentials to the windows credentials manager.
+func (h wincred) Add(creds *Credentials) error {
+	dp := winc.NewDomainPassword(creds.ServerURL)
```

Comment:
I think you want to use NewGenericCredential and not NewDomainPassword. The latter is used automatically to connect to other Windows machines e.g. for SMB shares and stuff.